src/Gui/ViewProviderDatum.cpp

```diff
@@ -95,31 +91,9 @@ void ViewProviderDatum::attach(App::DocumentObject* pcObject)
     matBinding->value = SoMaterialBinding::OVERALL;
     sep->addChild(matBinding);
 
-    // Scale feature to the given size
-    pScale->scaleFactor = SbVec3f(sz, sz, sz);
-    sep->addChild(pScale);
-
     // Setup font size
     auto font = new SoFont();
-    float fontRatio = 10.0f;
-    if (pcObject->is<App::Line>()) {
-        // keep font size on axes equal to font size on planes
-        fontRatio *= ViewProviderOrigin::axesScaling;
-        const char* axisName = pcObject->getNameInDocument();
-        auto axisRoles = App::Origin::AxisRoles;
-        if (strncmp(axisName, axisRoles[0], strlen(axisRoles[0])) == 0) {
-            // X-axis: red
-            ShapeAppearance.setDiffuseColor(0xFF0000FF);
-        }
-        else if (strncmp(axisName, axisRoles[1], strlen(axisRoles[1])) == 0) {
-            // Y-axis: green
-            ShapeAppearance.setDiffuseColor(0x00FF00FF);
-        }
-        else if (strncmp(axisName, axisRoles[2], strlen(axisRoles[2])) == 0) {
-            // Z-axis: blue
-            ShapeAppearance.setDiffuseColor(0x0000FFFF);
-        }
-    }
+    float fontRatio = 4.0f;
```

Comment:
Is this just arbitrary?

Reply:
Yes. It was like this before too. Though it was 10. 4 works better with the new style.